pages/index.vue

```diff
@@ -1,14 +1,68 @@
 <template>
-  <v-row justify="center" align="center">
-    <v-col cols="12" sm="8" md="6">
-      <v-card class="pa-5">
-        There is nothing, but soon something will be.
-      </v-card>
-    </v-col>
-  </v-row>
+<v-layout align-start justify-center>
```

Comment:
v-layout here doesn't require any authorization check, user without accessToken will see here form for adding wishes